sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuMultiFileReader.scala

```diff
@@ -308,6 +390,88 @@ case class CombineConf(
     combineThresholdSize: Long, // The size to combine to when combining small files
     combineWaitTime: Int) // The amount of time to wait for other files ready for combination.
 
+trait ThreadPoolConf {
```

Comment:
note, in future work we'll want to move this to a location that is separate from the readers. Not in this PR.

Reply:
added a TODO mark